Tests/Tracking/CustomerIOMockTest.swift

```diff
@@ -1,3 +1,4 @@
+@testable import CioInternalCommon
```

Comment:
```suggestion
```

I do understand why this import was added. Because `CustomerIOInstance` was moved into a different module, some parts of the codebase as well as customer's apps will be expected to find `CustomerIOInstance` and `CustomerIOInstanceMock` in Track module but will not be able to anymore. 

It's not ideal for a customer to need to have to import internal modules into their code base so to get around that, we have [these public aliases](https://github.com/customerio/customerio-ios/blob/a2ccebf1dae6d891c8c699005a083a4147241130/Sources/Tracking/Type/Aliases.swift). I anticipate you will need to add `CustomerIOInstance` and `CustomerIOInstanceMock` to that file, too.

Reply:
> some parts of the codebase as well as customer's apps will be expected to find CustomerIOInstance and CustomerIOInstanceMock in the Track module but will not be able to anymore.

I tried to cover it in this [comment](https://github.com/customerio/issues/issues/11645#issuecomment-1810153108) 

Why would a customer be using `CustomerIOInstance`? 
- We don't use it in our sample apps
- It's not part of our documentation
- None of the sample snippets ever shared with us include `CustomerIOInstance`
- It's not a general practice to use interface interface `CustomerIOInstance` instead of `CustomerIO` while adding an implementation. 

The only usage that we currently have is testing as well, so that makes me think, that maybe we are worrying about something customers aren't even possibly using. What are your thoughts on it?